feature-libs/asm/components/customer-emulation/customer-emulation.component.ts

```diff
@@ -25,12 +45,54 @@ export class CustomerEmulationComponent implements OnInit, OnDestroy {
     );
     this.isCustomerEmulationSessionInProgress$ =
       this.asmComponentService.isCustomerEmulationSessionInProgress();
+
+    this.cartId.valueChanges.subscribe((response: string) => {
+      this.cartIdExists = response.trim().length > 0;
+    });
+
+    this.activeCartFacade.getActiveCartId().subscribe((response) => {
+      if (response) {
+        this.cartId.setValue(response);
+      }
+    });
   }
 
   logoutCustomer() {
     this.asmComponentService.logoutCustomer();
   }
 
+  /**
+   * Assign the input cart number to the customer
+   */
+  assignCartToCustomer() {
+    const customerId = this.customer.uid;
+    const cartId = this.cartId.value;
+
+    if (customerId) {
+      this.occAsmAdapter.bindCart(cartId, customerId).subscribe(
+        () => {
+          this.showAssignCartSuccess = true;
+
+          setTimeout(() => {
+            this.showAssignCartSuccess = false;
+            this.submitEvent.emit({ customerId: customerId });
+          }, 3000);
+        },
+        () => {
+          this.showAssignCartError = true;
```

Comment:
There seems to be a bug in the case when the cart binding is refused by the backend. 
If I do so, and sign out completely the cs agent, I can't add anything to the cart once I'm back in the regular anon customer session.

Reply:
The issue seems resolved.